src/main/java/org/intellij/sonar/configuration/partials/SonarResourcesTableView.java

```diff
@@ -32,14 +33,18 @@
     @Nullable
     @Override
     public String valueOf(Resource sonarResource) {
-      if (Resource.QUALIFIER_PROJECT.equals(sonarResource.getQualifier())) {
-        return "Project";
-      } else
-        if (Resource.QUALIFIER_MODULE.equals(sonarResource.getQualifier())) {
+      SonarQualifier sonarQualifier = SonarQualifier.valueFrom(sonarResource.getQualifier());
+      if (sonarQualifier == null) {
+        return null;
+      }
+      switch (sonarQualifier) {
```

Comment:
here a yet another map like in the SonarQualifer enum is "better" then c style switch or if statements

Reply:
Ok, here it wasn't really me, I just improved an `if-else-if-else` to be a `switch` instead, but you are probably right, I change it to a `Map`.